dev-java/gradle-bin/metadata.xml

```diff
@@ -1,15 +1,12 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <maintainer type="person">
-    <email>chainsaw@gentoo.org</email>
-    <name>Tony Vroon</name>
-  </maintainer>
-  <maintainer type="person">
-    <email>flo@geekplace.eu</email>
-  </maintainer>
-  <maintainer type="project">
-    <email>java@gentoo.org</email>
-    <name>Java</name>
-  </maintainer>
+	<maintainer type="person">
```

Comment:
Once again you remove a lot of people from the metadata, have you talked with them beforehand? Do they approve this? Who's going to commit for you...